zt_frontend/src/components/FileExplorer.vue

```diff
@@ -49,6 +49,9 @@
               </v-btn>
             </template>
             <v-list>
+              <v-list-item v-if="item.file !== 'folder'" @click="openFileEditor(item)">
+                <v-list-item-title>Edit</v-list-item-title>
+              </v-list-item>
```

Comment:
Same as in the download PR, the dialogs should live here rather than making a call on click

Reply:
UPdated all dialogs UI and match the reuirements to be in v-list-item , added spinner and allowed folder upload and folder delete in upload functionality